dbscheme-proposal.md

```diff
@@ -0,0 +1,120 @@
+# Database Scheme
+
+## scheme
+* type (enc/sig)
```

Comment:
If we use strings for things like this (which would be fine with me), then we'd should properly define and  check the schema (e.g. https://github.com/cloudflare/json-schema-tools or https://www.npmjs.com/package/pajv should work).

Reply:
In general: yes. For this case, I've now started using subfolders for this. But probably we'll have this as an attribute on the internal level